docs/dapp/sapphire/quickstart.md

```diff
@@ -0,0 +1,395 @@
+# Quickstart
+
+In this tutorial, you'll port an Eth project in under 10 minutes (or your money
+back),
+and then go on to deploy a unique dapp that requires confidentiality to work.
+By the end of the tutorial, you should feel comfortable setting up your Eth
+development environment to target Sapphire, and know how and when to use
+confidentiality.
+
+The expected completion time of this tutorial is 30 minutes.
+
+## Port an Eth project
+
+### Setup
+
+Start by installing [Truffle](https://github.com/trufflesuite/truffle#install).
+
+Then, run these commands in your terminal:
+
+```sh
+mkdir MetaCoin && cd MetaCoin
+truffle unbox MetaCoin
+git init
+git add :/ && git commit -m "Initial commit"
+pnpm init && pnpm add -D @truffle/hdwallet-provider # npm or yarn also works
+```
+
+### Deploy to Emerald (non-confidential EVM)
+
+#### Get some eROSE
+
+:::tip
+
+To avoid confusion through the rest of this section we will denote ROSE the
+native token on the consensus layer only. When ROSE is deposited into Emerlad or
+Sapphire ParaTime we will call them eROSE or sROSE respectively.
+
+:::
+
+In order to deploy to the Emerald Testnet, we'll need some eROSE tokens.
```

Comment:
Now that I went through a complete tutorial, I would not use explicit eROSE and sROSE currencies, since there is only a simple faucet deposit involved.

```suggestion
#### Get some tokens on Emerald Testnet

In order to deploy to the Emerald Testnet, we'll need some tokens on our account first.
```

Reply:
This change will not be taken. This is the name people enter in metamask in the tutorial video. Also, we need a way to distinguish between the tokens, and we need to drill that in everywhere we can.